Sources/MessagesCollectionView.swift

```diff
@@ -36,6 +36,8 @@ open class MessagesCollectionView: UICollectionView {
 
     open weak var messageCellDelegate: MessageCellDelegate?
 
+    open weak var locationMessagesDisplayDelegate: LocationMessageDisplayDelegate?
```

Comment:
We don't need this property because we can assign it to `messagesDisplayDelegate` since `LocationMessageDisplayDelegate` conforms to `MessagesDisplayDelegate`

Reply:
Got it! Thanks!